circle.yml

```diff
@@ -0,0 +1,16 @@
+dependencies:
+  cache_directories:
+    - /usr/local/Homebrew
+  override:
+    - brew install shellcheck
+
+machine:
```

Comment:
it's containerized with docker, so you can add custom containers easily. probably not important for this repo. not sure how long 1.0 will be around, though